proxy/src/cache/endpoints.rs

```diff
@@ -71,14 +72,17 @@ impl EndpointsCache {
         }
         // If cache is disabled, just collect the metrics and return.
         if self.config.disable_cache {
-            ctx.set_rejected(self.should_reject(endpoint));
+            let rejected = self.should_reject(endpoint);
+            ctx.set_rejected(rejected);
+            info!(?rejected);
```

Comment:
can you include a searchable message in this, `"check endpoint is valid"`, and then also include a variable in this one to indicate that the action was not taken as it's disabled?